dcos_installer/config_util.py

```diff
@@ -23,6 +23,8 @@ def do_configure(config):
     # Write some package metadata
     pkgpanda.util.write_json(CLUSTER_PACKAGES_PATH, gen_out.cluster_packages)
 
+    return gen_out
```

Comment:
Nit: rename method from `do_configure` to something more appropriate to signify the behavior change of it now returning something.